CHANGELOG.rst

```diff
@@ -6,6 +6,65 @@
 Synapse Changelog
 *****************
 
+v2.198.0 - 2025-02-14
+=====================
+
+Automatic Migrations
+--------------------
+- Added a migration to lock any service users which are marked as archived but
+  are not locked.
+  (`#4100 <https://github.com/vertexproject/synapse/pull/4100>`_)
+- See :ref:`datamigration` for more information about automatic migrations.
+
+Model Changes
+-------------
+- See :ref:`userguide_model_v2_198_0` for more detailed model changes.
+
+Features and Enhancements
+-------------------------
+- Added brief IO yields for various LMDB operations to help improve
+  responsiveness when handling certain transactions.
+  (`#3937 <https://github.com/vertexproject/synapse/pull/3937>`_)
+- Added new endpoints section in the package definition schema, allowing the
+  configuration of multiple endpoints and details.
+  (`#4127 <https://github.com/vertexproject/synapse/pull/4127>`_)
+- Added ``++=``, ``--=``, ``?++=``, and ``?--=`` edit operators to Storm for
+  adding or removing multiple values from an array property in a single
+  operation.
+  (`#4130 <https://github.com/vertexproject/synapse/pull/4130>`_)
+- Updated Storm lift behavior to allow lifting guid forms and properties using
+  the property dictionary syntax.
+  (`#4132 <https://github.com/vertexproject/synapse/pull/4132>`_)
+
+Bugfixes
+--------
+- Fixed bug where archived users could be unlocked.
+  (`#4100 <https://github.com/vertexproject/synapse/pull/4100>`_)
+- Fixed an issue in ``$lib.regex`` where Python regex exceptions were leaking
+  into the Storm runtime.
+  (`#4118 <https://github.com/vertexproject/synapse/pull/4118>`_)
+- Fixed bug with ``syn:user`` and ``syn:role`` nodes when queried name does not
+  exist and the value is not a valid ``guid``.
```

Comment:
```suggestion
  exist and the value is not a valid guid.
```

Reply:
We typically quote types, so this was a valid use case.

Reply:
There's a "guid" lower in the changelog then that needs to be quoted.